app/controllers/observations_controller/new_and_create.rb

```diff
@@ -67,9 +67,16 @@ def defaults_from_last_observation_created
     %w[when where location is_collection_location gps_hidden].each do |attr|
       @observation.send("#{attr}=", last_observation.send(attr))
     end
-    last_observation.projects.each do |project|
-      @project_checks[project.id] = true unless project.open_membership
-    end
+
+    # FIXME: Why woold open-membership projects be treated different from
+    # closed-membership projects for check-box persistence?
+    last_observation.projects.where(open_membership: false).
+      find_each do |project|
+        next unless project.current?
+
+        @project_checks[project.id] = true
+      end
+
```

Comment:
@mo-nathan 
See FIXME and comment above about related FIXME.
Note that I did not change the functionality here. (I forget whey. Probably easier because the change broke some tests.)
I would like to change the functionality so that project checkboxes persist identically for open- and closed- membership projects.